src/components/NodesList/index.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import getNodesList from '../../redux/actions/nodeClusterActions';
+// import tellAge from '../../helpers/ageUtility';
+import './NodesList.css';
+import NavBar from '../NavBar';
+import Status from '../Status';
+import SpinnerComponents from '../SpinnerComponent';
+import InformationBar from '../InformationBar';
+import SideNav from '../SideNav';
+
+
+class NodesList extends Component {
+  componentDidMount() {
+    const { getNodesList } = this.props;
+    const { match: { params } } = this.props;
+    getNodesList(params.clusterID);
+  }
+
+  getAge(utcDate) {
+    const creationTimestamp = new Date(utcDate).getTime();
+    return creationTimestamp;
+  }
+
+  getRoles(node) {
+    if (typeof (node.spec.taints) !== 'undefined') {
+      const str = String(node.spec.taints[0].key);
+      // get strings after /
```

Comment:
Kindly explain this comment.

Reply:
`str` gets a long string and am interested in the role string which is after a `/`. So the next line returns all characters after `/`

Reply:
@Mubangizi thanks